resources/dialog.ui

```diff
@@ -532,7 +532,7 @@ image sequences</string>
               <item>
                <widget class="QStackedWidget" name="details_stack">
                 <property name="currentIndex">
-                 <number>1</number>
+                 <number>0</number>
```

Comment:
Is this on purpose? Does it cause any flicker in the GUI?

Reply:
No its not on purpose, and I didn't notice any flicker, I suspect it switches back before drawn. However I'll update the UI to revert this change anyway, good catch!